spl/src/metadata.rs

```diff
@@ -1,5 +1,6 @@
 use anchor_lang::context::CpiContext;
 use anchor_lang::{Accounts, Result, ToAccountInfos};
+use mpl_token_metadata::deser::meta_deser;
```

Comment:
Can be dropped as import as called through Metadata::deserialize now.

Reply:
Change is a bit incomplete right now but the idea is to move to the new function, the rust package for the metadata program hasn't been updated yet so I'm waiting on this @samuelvanderwaal

Reply:
@kevinrodriguez-io mpl-token-metadata 1.3.2 with the deserialize_safe create is released.

https://crates.io/crates/mpl-token-metadata

I think you are good to adjust this PR.

Reply:
Awesome, I'll be updating it today ! Thanks